### DISCUSSION TITLE
"Email is missing" error

### DISCUSSION THREAD

**Discussion by satbav** [harikrishna43/UmbracoForms.Mailchimp.WorkFlow#14] [General]
HI,

Not sure if this plugin is still supported. I've set it up, and entered the correct audience ids and api keys.
However, from the logs I can see that I always get a "Email is missing".

I've added a field, textbox, called Email, alias: "email".

I've mapped the field as instructed:

![image](https://user-images.githubusercontent.com/6164960/109854859-75782200-7c4f-11eb-8cd6-724a4d7ad09d.png)

Any idea what might be the issue?

Thanks

**Comment by harikrishna43** [comment]
What Umbraco version are you using?
